literature/literature.MD

```diff
@@ -0,0 +1,18 @@
+# Books to get started with FastAPI and microservice architecture
+First, get access to the [eBook library](https://ebookcentral.proquest.com/auth/lib/mosbach-dhbw/).
+
+## FastAPI
+[Building Data Science Applications with FastAPI](https://ebookcentral.proquest.com/lib/mosbach-dhbw/detail.action?docID=6727091)
+
+Mainly `Chapter 3: Developing a RESTful API with FastAPI` (page 61 - 112)
+
+
+## Microservice architecture
+
+General information: [Microservice Patterns and Best Practices](https://ebookcentral.proquest.com/lib/mosbach-dhbw/detail.action?docID=5259455&query=microservice+python)
+
+[Python Microservices Development](https://ebookcentral.proquest.com/lib/mosbach-dhbw/detail.action?docID=6739164)
+: Take a look on `Chapter 1: Understanding Microservices`
+
+[Python Microservices Development](https://ebookcentral.proquest.com/lib/mosbach-dhbw/detail.action?docID=4998695)
```

Comment:
This line seems to be a duplicate from line 14.

Reply:
Actually this are two different books, but they deal both about Microservices in Python, to understand the basics of microservice architectures.

Reply:
Do not reference to books in this way. Please provide information about the author, ISBN and so on.